CNAME

```diff
@@ -1 +0,0 @@
-widget.bikeindex.org
```

Comment:
Not sure where it belongs in the eventual process, whether it goes in the `gh-pages` branch or in `master` - but I think we'll need this? Or has github pages changed how they manage this?

Reply:
I think we can actually keep it here.  I added it back in - thanks!